brainglobe_atlasapi/atlas_generation/atlas_scripts/kim_devccf_mouse.py

```diff
@@ -0,0 +1,263 @@
+
+    labels = np.unique(annotation_volume).astype(np.uint16)
+
+    for key, node in tree.nodes.items():
+        if key in labels:
+            is_label = True
+        else:
+            is_label = False
+        node.data = Region(is_label)
+
+    # Mesh creation
+    closing_n_iters = 2
+    decimate_fraction = 0.2  # 0.04
+    # What fraction of the original number of vertices is to be kept.
+    smooth = False  # smooth meshes after creation
+    start = time.time()
+    if PARALLEL:
+        pool = mp.Pool(min(mp.cpu_count() - 2, 16))
+        try:
+            pool.map(
+                create_region_mesh,
+                [
+                    (
+                        meshes_dir_path,
+                        node,
+                        tree,
+                        labels,
+                        annotation_volume,
+                        root_id,
+                        closing_n_iters,
+                        decimate_fraction,
+                        smooth,
+                    )
+                    for node in tree.nodes.values()
+                ],
+            )
+        except mp.pool.MaybeEncodingError:
+            pass
+    else:
+        for node in track(
+            tree.nodes.values(),
+            total=tree.size(),
+            description="Creating meshes",
+        ):
+            # root_node = tree.nodes[root_id]
+            create_region_mesh(
+                (
+                    meshes_dir_path,
+                    # root_node,
+                    node,
+                    tree,
+                    labels,
+                    annotation_volume,
+                    root_id,
+                    closing_n_iters,
+                    decimate_fraction,
+                    smooth,
+                )
+            )
+
+    print(
+        "Finished mesh extraction in: ",
+        round((time.time() - start) / 60, 2),
+        " minutes",
+    )
+    return meshes_dir_path
+
+
+def create_mesh_dict(structures, meshes_dir_path):
+    meshes_dict = dict()
+    structures_with_mesh = []
+    for s in structures:
+        # Check if a mesh was created
+        mesh_path = meshes_dir_path / f'{s["id"]}.obj'
+        if not mesh_path.exists():
+            print(f"No mesh file exists for: {s}, ignoring it")
+            continue
+        else:
+            # Check that the mesh actually exists (i.e. not empty)
+            if mesh_path.stat().st_size < 512:
+                print(f"obj file for {s} is too small, ignoring it.")
+                continue
+        structures_with_mesh.append(s)
+        meshes_dict[s["id"]] = mesh_path
+    print(
+        f"In the end, {len(structures_with_mesh)} "
+        "structures with mesh are kept"
+    )
+    return meshes_dict, structures_with_mesh
+
+
+if __name__ == "__main__":
+    atlas_root = "/home/castoldi/Downloads/DevCCFv1"
+    # Generated atlas path:
+    DEFAULT_WORKDIR = Path.home() / "brainglobe_workingdir"
```

Comment:
Once we figure out how to download the data, we should avoid this hard-coded path (and ideally replace it with a `pooch` temporary file)

Also, you can import the DEFAULT_WORKDIR
```python
from brainglobe_atlasapi.config import DEFAULT_WORKDIR
```
(so we can change it with one-line in the future!)